src/csvToJson.js

```diff
@@ -12,20 +12,17 @@ import { joinPath } from './utils';
 // is there some intersections, etc.
 // I think we can improve it very easy.
 
-const maxEntries = 10000;
-let result = [];
+
 let folderName;
-let numberOfFiles;
 
-const generateJsonFiles = (i, fileName, start, stop) => {
-  const data = result.slice(start, stop);
+
+const generateJsonFiles = (i, fileName, start, stop, dataEntries) => {
+  const data = dataEntries.slice(start, stop);
 
   // @TODO change that
   // we can also create a method for path.join, so it wouldn't complicate our code
   // really bad line
-  const jsonFileName = `${folderName}/${fileName}${i}.json`;
-  // Why use USFA when jsonFileName already has the folderName in it.
```

Comment:
please be very careful with comments, I mean it better to file a separated issue and discuss something, rather than just delete some up-perfect things and they will be gone